### DISCUSSION TITLE
SCP S-100 machine - how to start it?

### DISCUSSION THREAD

**Discussion by Torinde** [dbalsom/marty#124] [Q&A] [answered]
It seems the ROM files for "s100" are not in the release package and I can't find "MON_1.5_1982-06-10_CROMEMCO4FDC.BIN" - should I get it from MAME?

I want to try some of the available images for QDOS/86-DOS ([SCP](https://winworldpc.com/product/86-dos/100), [NSI](http://www.bitsavers.org/bits/SeattleComputerProducts/NSI/)) and there are such for SCP Cromemco 4FDC (and SCP Tarbell and North Star).

**Comment by dbalsom** [comment]
There is no actual S-100 machine or hardware support.  The S-100 ROM definition I added was just for an experiment to boot the SCP 8086 ROM monitor code and see what it did.

**Reply by dbalsom** [reply, marked as answer]
that file wasn't actually supposed to be in the release :P  Oops.

**Reply by Torinde** [reply]
Clear, but then for the reason above - I suggest it as a feature :)

Source code for the controllers (https://github.com/86Box/86Box/discussions/3857#discussioncomment-9844218):
https://github.com/simh/simh/blob/master/AltairZ80/s100_tarbell.c
https://github.com/simh/simh/blob/master/AltairZ80/s100_64fdc.c

Related: https://github.com/joncampbell123/dosbox-x/discussions/5062#discussion-6846770

**Reply by dbalsom** [reply]
The SIMH source tree is poisoned by some unfortunate drama surrounding incompatible license changes by the maintainer.
The Open-SIMH project is the continuation of the original open-source effort.
https://github.com/open-simh

That said, I generally do not borrow code from other emulators or projects.  I prefer to implement things directly from documentation or hardware research.

**Reply by Torinde** [reply]
Great, here is some documentation about [SCP200B](http://www.s100computers.com/Hardware%20Folder/Seattle%20Computer%20Products/8086%20CPU%20Board/8086%20Board.htm).
- ["ROM Monitor"](https://github.com/TheBrokenPipe/SCP-8086-Monitor), [YASM port](https://github.com/lkundrak/scp-8086-monitor.yasm)

**Reply by Torinde** [reply]
It seems SCP200B supported disabling the boot ROM after boot-up, which allows for DOS to utilize the whole 1MB as Conventional memory: Microsoft were using that for "[linking the linker](https://en.wikipedia.org/wiki/Conventional_memory#cite_note-Paterson_2007_SCP-13)" until 386 came.
